app/script/search/SearchRepository.js

```diff
@@ -67,44 +66,69 @@ z.search.SearchRepository = class SearchRepository {
    *
    * @param {string} term - the search term
    * @param {Array<z.entity.User>} userEntities - entities to match the search term against
-   * @param {Array<z.search.SearchRepository.CONFIG.SEARCHABLE_FIELDS>} properties=['first_name', 'last_name', 'username'] - list of properties that will be matched against the search term
+   * @param {Array<z.search.SearchRepository.CONFIG.SEARCHABLE_FIELDS>} properties=['name', 'username'] - list of properties that will be matched against the search term
```

Comment:
The jsdoc should also contain the constants

Reply:
I would keep them here (even if they are not in the signature) because we set them programmatically now but the behavior stays exactly the same from the user point of view